_usage/mvu.md

```diff
@@ -0,0 +1,156 @@
+
+If you are running version 1.3.1, use the following steps to perform an MVU on a Babelfish server:
+
+1. Follow the [instructions at the Babelfish repo](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_2_X_DEV/contrib/README.md#build-the-postgres-engine) to build and install the Babelfish version 2.2.0 PostgreSQL server and extensions.
+
+2. Then, initialize the `data` directory for new engine version. For convenience, add /opt/mssql-tools/bin/ and ~/postgres/bin to your PATH environment variable. This enables you to run the tools without specifying the full path. For Z shell (the default shell for Cloud Desktop), add the following lines to your ~/.zshrc file:
+
+```sql
+export PATH=/opt/mssql-tools/bin:$PATH
+export PATH=~/postgres/bin:$PATH
+```
+
+3. Adjust the paths of the following commands to point to the directory of the target server, and run them to start the Postgres server:
+
+```sql
+~/postgres/bin/initdb -D ~/postgres/data/
+~/postgres/bin/pg_ctl -D ~/postgres/data/ -l logfile start
+```
+
+4. Modify the `~/postgres/data/postgresql.conf` file by uncommenting the following parameters and including the `babelfishpg_tds` extension in the `shared_preload_libraries` parameter:
+
+```sql
+listen_addresses = '*'
+shared_preload_libraries = 'babelfishpg_tds'
+```
+
+5. Modify `~/postgres/data/pg_hba.conf` to [allow connections](https://www.postgresql.org/docs/14/auth-pg-hba-conf.html) from your IP address:
+
+```sql
+host    all             all     10.x.y.z/32            trust
+```
+
+6. Use the following command to apply the changes:
+
+```sql
+~/postgres/bin/pg_ctl -D ~/postgres/data/ -l logfile restart
+```
+
+Note that new server should not contain a Babelfish database, or have the extensions installed.
+
+7. Use `pg_ctl` to stop both `source` and `target` servers:
+
+  On the old server:
+
+```sql
+  ~/postgres_old/bin/pg_ctl -D ~/postgres_old/data/ stop
+```
+
+  On the new server:
+
+```sql  
+  ~/postgres_new/bin/pg_ctl -D ~/postgres_new/data/ stop
+```
+
+8. Then, use the `pg_upgrade` utility to perform the major version upgrade.  For detailed information about using pgupgrade, refer to the [official postgres documentation](https://www.postgresql.org/docs/current/pgupgrade.html). For example:
+
+```sql
+mkdir ~/upgrade
+cd ~/upgrade
+~/postgres_new/bin/pg_upgrade -U $USER -b ~/postgres_old/bin \
+                                       -B ~/postgres_new/bin \
+                                       -d ~/postgres_old/data \
+                                       -D ~/postgres_new/data \
+                                       -p 5432 -P 5433 -j 4 --link --verbose
+``` 
+
+9. When `pg_upgrade` completes, start the new server:
```

Comment:
I have added steps to follow in case of failure in internal doc, please take a look at them and incorporate those steps here as well.

Reply:
@rishabhtanwar29 

Can you tell me what we need to add from the internal doc for the OSS users please?